### DISCUSSION TITLE
Video Loaded:  How do I switch between Audio Tracks?

### DISCUSSION THREAD

**Discussion by Davexx** [SubtitleEdit/subtitleedit#6389] [Q&A] [answered]
Hello,
I have a video file with two audio tracks and I want to use the 2nd audio track to sync the subtitles. 
How do I change from the default 1st audio track to the second audio track?
I cannot find that option anywhere.

Thanks Davexx

**Comment by LeonCheung** [comment, marked as answer]
If there really are multiple tracks, you'll see the option via Video > Choose audio track...

![image](https://user-images.githubusercontent.com/494788/200211039-a37f6353-d1fe-4105-aa58-2b238d5348d5.png)

**Reply by Davexx** [reply]
Hmmm, I don't seem to have that option, yet the video MKV video file contains two audio tracks:
![image](https://user-images.githubusercontent.com/3362914/200211352-b962e2cc-a389-456a-bb86-da4776a24667.png)

**Reply by LeonCheung** [reply]
Then please make sure it's a video with multiple tracks. You'll be given the option when trying to generate the waveform:

![image](https://user-images.githubusercontent.com/494788/200211568-1f5983f3-92c8-4ac6-9553-2c4618f83443.png)

If you got mpv installed, play the video with mpv, and check how many tracks are there:

![image](https://user-images.githubusercontent.com/494788/200211803-7e23cc5e-64e1-45d1-afda-96d160712616.png)

**Reply by Davexx** [reply]
I do not have mpv installed.  I assume that is a plugin?

**Reply by LeonCheung** [reply]
https://mpv.io/

**Reply by Davexx** [reply]
Oh I see, the default video player is bare bones, but you can add other video players that extend the functionality, like mpv, which will allow you to change the audio track, correct?

**Reply by LeonCheung** [reply]
It is the most recommended video engine in SE so far.  Personally, it's my first choice. :)
![image](https://user-images.githubusercontent.com/494788/200212339-77609ef9-714a-4db2-87df-c178545f5eb3.png)

**Reply by Davexx** [reply]
Oh, Splendid! 
Thank you so much for the recommendation.

**Comment by Davexx** [comment]
I figured out a way to reset it if you are using the default video player: 
Go to the **SubtitleEdit/Waveforms** Folder and delete the audio wave files,
 then: 
New > Click to Add Waveform
Then rechoose audio track by selecting the second option.
![image](https://user-images.githubusercontent.com/3362914/200211667-cc342b8b-f8f0-4582-a3b2-f77aa68940d9.png)
